packages/forma-36-website/src/pages/guidelines/accessibility/index.mdx

```diff
@@ -0,0 +1,64 @@
+
+### Text Alternatives
+
+- All non-text content should provide alternative text (e.g. `alt` attribute for images, `label` or `describedby` aria attribute)
+
+### Adaptable
+
+- Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text. E.g. Provide labels for form components or use the aria `describedby` attribute to create relationship between content.
+- Order content in a meaningful sequence. Use headings to structure content sections.
+
+### Keyboard Accessibility
+
+- Build for keyboard use only, all interactive elements should be focusable and show focus indicators like outlines or other stylings to guide the interaction.
+- Prevent keyboard traps. Every element that can be focused on using the keyboard can be unfocused using the keyboard.
+- If character key shortcuts are implemented (e.g. shortcuts just using a letter, number or symbol) it must be possible to either disable or reassign them.
+
+### Distinguishable
+
+- Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element. E.g. A status like published, archived or deleted should not only be indicated by its color coding but show an additional status text.
+- Respect the contrast minimum. Paragraph text has a contrast of at least 4:5:1, the visual presentation of user interface or graphical components has a minimum contrast of 3:1 ([https://webaim.org/resources/contrastchecker/](https://webaim.org/resources/contrastchecker/))
+- Text can be resized up to 200 percent without loss of content or functionality (Except for captions and images of text)
+
+### Navigable
+
+- Provide a mechanism like skip links to bypass blocks of repeated content
+- All Pages have titles that describe the purpose of the page
+- Elements of the page can be focused in the sequence of appearance
+- The purpose of each link can be determined by the link text alone
+
+The WCAG provides a quick reference to all Level AA success criteria which can be found here:
+
+[https://www.w3.org/WAI/WCAG21/quickref](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa&techniques=advisory%2Cfailures&technologies=flash%2Csl&showtechniques=133#top)
+
+### Automated Accessibility Testing
+
+- When implementing Forma 36 react components it is important to provide a basic level of automated testing for new components. This is why we use AXE and Jest to test component snapshots on WCAG criteria. Every component bootstrapped using the CLI (e.g. `yarn add-component`) will provide a basic Jest test for accessibility of the component snapshot. More information about testing can be found here: [https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/README.md#testing](https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/README.md#testing)
```

Comment:
Perhaps it would be useful to link to the AXE and Jest Github repositories because there doesn't seem to be a mention of AXE in the testing information README.